### PR TITLE
Bump version

### PR DESCRIPTION
**Pull request by lukabratos**
### What?

Release version 0.10.12

----
CC @pusher/mobile

### CODE REVIEW

File: Sources/Array+CalculateMD5Hash.swift [hunk 1]
```diff
@@ -1,20 +1,10 @@
...
-        var digestData = Data(count: Int(CC_MD5_DIGEST_LENGTH))
-
-        _ = digestData.withUnsafeMutableBytes {digestBytes in
-            messageData.withUnsafeBytes {messageBytes in
-                CC_MD5(messageBytes, CC_LONG(messageData.count), digestBytes)
-            }
-        }
-
-        return digestData.map { String(format: "%02hhx", $0) }.joined()
+        return MD5(elementsJoined)
```
Comment (luismfonseca) [inline Sources/Array+CalculateMD5Hash.swift]: So, can this MD5 implementation fail?

Can this function signature instead be `calculateMD5Hash() -> String`?

File: Sources/MD5.swift [hunk 2]
```diff
@@ -0,0 +1,274 @@
+/*
```
Comment (luismfonseca) [inline Sources/MD5.swift]: What's the advantage of bring this in vs using a library?

Comment (lukabratos) [reply Sources/MD5.swift]: I don't understand this question completely. Which library did you have in mind?

File: Sources/Array+CalculateMD5Hash.swift (hunk 1 above)
Comment (lukabratos) [reply Sources/Array+CalculateMD5Hash.swift]: >So, can this MD5 implementation fail?

There are a few tests that validate [implementation](https://github.com/onmyway133/SwiftHash/blob/master/SwiftHashTests/Tests.swift) of MD5. I don't have exact answer because I didn't test it throughoutly for every single scenario/edge case that could potentially happen. It's the same case with all third party dependencies ... Ideally we would use [CommonCrypto](https://opensource.apple.com/source/CommonCrypto/) from Apple but there's a lot of [issues](https://stackoverflow.com/questions/25248598/importing-commoncrypto-in-a-swift-framework) and no elegant solutions in order to make it work in a Swift framework (this was resolved in Xcode 10 but we still need to support Xcode 9).

>Can this function signature instead be calculateMD5Hash() -> String?

In `calculateMD5Hash` function I'm also first sorting the elements in the array and then joining them together in order to calculate the MD5 hash. By using just `MD5` function I would need to move this logic elsewhere and I don't see a point of doing that.

File: Sources/MD5.swift (hunk 2 above)
Comment (luismfonseca) [reply Sources/MD5.swift]: Can you add that last sentence as a code comment in the top of this file? That makes the motivation clearer without needing to search github/commit messages.
